### PR TITLE
Renamed ProtectedAttribute to Ladon

### PR DESCRIPTION
**Pull request by mvgijssel**
We needed a rename for the gem, because the name was singular. And when changing the name, why not choose something REALLY cool like "Ladon", which also gives it a neat back story:

> Ladon was the serpent-like dragon that twined and twisted around the tree in the Garden of the Hesperides and guarded the golden apples.

And because we're not perfect and there are excellent Hackers on the platform:
> He was overcome by Heracles.

https://en.wikipedia.org/wiki/Ladon_(mythology)

Not the forget, the name needs to be available on Rubygems. So already took the liberty of publishing 0.1.0 at https://rubygems.org/gems/ladon

### CODE REVIEW

File: ladon.gemspec [hunk 1]
```diff
@@ -25,15 +24,6 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/hacker0x01/protected_attributes"
   spec.license       = "MIT"
 
-  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
```
Comment (mvgijssel) [inline ladon.gemspec]: This section needs to be removed to be able to publish to Rubygems (like the comment says :p)

**Comment by Willianvdv** [comment]
@alexanderjeurissen I think it's a bit silly to insta merge. You didn't give a fair chance to respond to this PR.

**Comment by alexanderjeurissen** [comment]
@Willianvdv Seeing as I didn't author this PR and the fact that this PR only proposes a name change (which I discussed with @mvgijssel offline) I didn't expect this change to be controversial / needing further discussion.

If you feel that this change is undesirable I'm more then happy to oblige and revert this commit and restore the branch.

**Comment by Willianvdv** [comment]
@alexanderjeurissen yes, revert it!

**Pull request merged**